ui/ducks/send/send.js

```diff
@@ -1585,6 +1609,7 @@ export function updateRecipient({ address, nickname }) {
         nickname: nickname || nicknameFromAddressBookEntryOrAccountName,
       }),
     );
+    await dispatch(qeditSecurityCheckStatus());
```

Comment:
not sure I understand this, would love to sit with you so that you explain to me this part